proto/ha_set.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package dash.ha_set;
+
+import "types.proto";
+
+enum Owner{
```

Comment:
same as Role in ha scope, maybe better to move to a shared place and give it a more unique name, such as HaSetOwner and etc.